tests/ui/cfg/true-false.rs

```diff
@@ -0,0 +1,29 @@
+//@ run-pass
+
+#![feature(link_cfg)]
+
+#[cfg(true)]
+fn foo() -> bool {
+    cfg!(true)
+}
+
+#[cfg(false)]
+fn foo() -> bool {
+    cfg!(false)
+}
+
+#[cfg_attr(true, cfg(false))]
+fn foo() {}
+
+#[link(name = "foo", cfg(false))]
+extern "C" {}
+
+fn main() {
+    assert!(foo());
```

Comment:
Should this be `!foo()`?

Reply:
No, `foo()` does return `true` from the first definition:
```
#[cfg(true)]
fn foo() -> bool {
    cfg!(true) // this return true
}
```